tests/Integration/WorkloadManagementSystem/Test_JobParameters_MySQLandES.py

```diff
@@ -33,7 +33,7 @@ def createJob():
 
   wmsClient = WMSClient()
   res = wmsClient.submitJob(job._toJDL(xmlFile=jobDescription))
-  assert res['OK']
+  assert res['OK'], res['Message']
```

Comment:
Would it be better to configure pytest to show this rather than adding lots of boilerplate?

```bash
$ pytest --showlocals
===================================================================== test session starts ======================================================================
platform darwin -- Python 3.8.1, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /private/tmp/check-pytest
plugins: error-for-skips-2.0.2, celery-4.4.0, cov-2.8.1
collected 1 item

test_something.py F                                                                                                                                       [100%]

=========================================================================== FAILURES ===========================================================================
________________________________________________________________________ test_something ________________________________________________________________________

    def test_something():
        res = {"OK": False, "Message": "asdsadasda"}
>       assert res["OK"]
E       assert False

res        = {'Message': 'asdsadasda', 'OK': False}

test_something.py:3: AssertionError
====================================================================== 1 failed in 0.07s =======================================================================
```